x-pack/platform/plugins/shared/index_management/public/application/sections/home/index_list/details_page/convert_to_lookup_index_modal/convert_to_lookup_index_modal.tsx

```diff
@@ -97,16 +107,23 @@ export const ConvertToLookupIndexModal = ({ onCloseModal, onConvert, sourceIndex
       </EuiModalHeader>
 
       <EuiModalBody>
+        <FormattedMessage
+          id="xpack.idxMgmt.convertToLookupIndexModal.modalBodyDescription"
+          defaultMessage="Converting an index to a Lookup index will create a new index with Lookup mode set. The old index will continue to exist."
```

Comment:
```suggestion
          defaultMessage="A new lookup index will be created, and the original index will still exist as before."
```
Shorter, straight to the point